app/screens/sso/sso.js

```diff
@@ -85,6 +95,25 @@ class SSO extends PureComponent {
         });
     };
 
+    onMessage = (event) => {
+        try {
+            const response = JSON.parse(event.nativeEvent.data);
```

Comment:
Are we always going to have a data property here? `JSON.parse` will throw if it's null or undefined right?